administration/src/bp-modules/cards/ImportCardsController.tsx

```diff
@@ -59,23 +59,15 @@ const InnerImportCardsController = ({ region }: { region: Region }): ReactElemen
     }
   }
 
+  // TODO headers or csvHeader?
   const lineToBlueprint = useCallback(
-    (line: string[], csvHeader: string[]): CSVCard => {
+    (line: string[], csvHeader: string[]): CardBlueprint => {
       if (isFreinetFormat) {
         convertFreinetImport(line, csvHeader, projectConfig)
       }
-      const cardBlueprint = new CSVCard(projectConfig.card, region)
-      headers.forEach(header => {
-        const idx = csvHeader.indexOf(header)
-        if (idx === -1) {
-          // column is missing in csv
-          return
-        }
-        cardBlueprint.setValue(header, line[idx])
-      })
-      return cardBlueprint
+      return initializeCardFromCSV(projectConfig.card, line, csvHeader, region)
```

Comment:
This is still really confusing, why are haders and csvHeader mixed here? Which one do I have to use?

Reply:
we have headers defined in the config  (headers) and the headers that were read from the csv (csvHeaders).

Reply:
I will create a refactoring issue as this is really confusing...

Reply:
https://github.com/digitalfabrik/entitlementcard/issues/1700